### ISSUE TITLE
cannot lastest version on angular 18 without using --force

### ISSUE THREAD

**Issue by kelvinZoe** [ngxpert/hot-toast#20]
Please can you provide a compatible version for angular 18? I heavily depend on it to handle my errors on my application. The only workaround was using --force. I would be sooo happy to have it made soon. Looking forward to this fix soon. Thanks

**Comment by shhdharmen** [comment]
What error you're getting when doing the installation without --force?

**Comment by kelvinZoe** [comment]
thanks for responding so quickly.
-my node version is <code>Node.js v20.16.0</code>
-my npm version `10.8.1`
-my angular version 
<img width="495" alt="Screenshot 2024-09-06 at 2 16 53 PM" src="https://github.com/user-attachments/assets/c9440f21-0e3d-4ae4-adc2-db9cd694778f">
When I do the install without the force...
<img width="814" alt="Screenshot 2024-09-06 at 2 21 12 PM" src="https://github.com/user-attachments/assets/e55d7c23-043f-42f0-a86c-41518c042566">

I just realized my own fault, I had updated my angular cli to 18 on a different project and not this particular one. After updating the cli to the lastest version which is 18.2.3, it worked.

I have a question tho, is this repo the same as ngneat? I think that was the version I was using but couldnt install on the angular 18 evn.
